app/assets/stylesheets/homepage.scss

```diff
@@ -1,3 +1,27 @@
+//========= bootstrap scrollspy on landing page ==============
+
+body {
+	position: relative;
+}
+ul.nav-pills {
+	top: 40%;
+	left: 95%;
+	position: fixed;
+	z-index: 10;
+}
+.nav-pills > li.active > a, .nav-pills > li.active > a:hover, .nav-pills > li.active > a:focus {
+	background-color: #404040;
+}
+.nav-pills > li > a {
+	border-radius: 50%;
+	height: 29px;
+	margin-top: 8px;
+}
+a {
```

Comment:
Un-nested rules like this will affect <a> elements throughout the entire site, so I'm going to nest this in a bit.